packages/@tests/smoke/web-cra-ts/test/index.ts

```diff
@@ -2,19 +2,24 @@ import puppeteer from 'puppeteer';
 import { dirname, join } from 'path';
 import { fileURLToPath } from 'url';
 
-import { startCdn, startContentServer, stopServer } from '@test/test-utils';
+import { CDN_PUBLIC_PATH, startContentServer, stopServer } from '@test/test-utils';
+import { access, symlink } from 'fs/promises';
 
 const port = 3001;
 const uri = `http://localhost:${port}/`;
 const __dirname = dirname(fileURLToPath(import.meta.url));
 const publicPath = join(__dirname, '../build/');
 
 const test = async () => {
-    const cdn = await startCdn(8766);
     const localServer = await startContentServer(port, publicPath);
+    try {
+        await access(join(publicPath, 'source'))
+    } catch {
+        await symlink(CDN_PUBLIC_PATH, join(publicPath, 'source'));
+    }
 
     console.log('starting puppeteer...');
-    const browser = await puppeteer.launch();
+    const browser = await puppeteer.launch({ headless: false });
```

Comment:
If I am understanding correctly you used that temporarely to check something. If so - please don't forget to remove it

Reply:
You didn't answer anything to my comment. Will this run in non-headless mode in CI if you leave it like this?